RebrickableSharp.Client.Csv/JsonCsvClassMap.cs

```diff
@@ -0,0 +1,44 @@
+﻿using CsvHelper.Configuration;
+using System.Reflection;
+using System.Text.Json.Serialization;
+
+namespace RebrickableSharp.Client.Csv;
+
+/// <summary>
+/// Helper class to generate a Csv ClassMap from JsonPropertyName attributes.
+/// </summary>
+/// <typeparam name="T"></typeparam>
+public class JsonCsvClassMap<T> : ClassMap<T> where T : class

```

Comment:
I guess this class can be made internal?

Reply:
Probably yes. I'll do another pass at different modifiers.

Reply:
c70e5eb6030008856b27a01a6ee63290cfea2d73